docs/scripting/resources/gametextstyles.md

```diff
@@ -0,0 +1,82 @@
+---
+title: GameText Styles
+
+description: GameText Styles to be used in textdraws and gametext
+---
+
+To be used with [GameText](https://wiki.open.mp/docs/scripting/functions/GameTextForPlayer) and [textdraws](https://wiki.open.mp/docs/scripting/functions/TextDrawCreate).
+
+## `Text Colors`
+* `~n~` New line
+
+* `~r~` Red
+
+* `~g~` Green
+
+* `~b~` Blue
+
+* `~w~` or ~s~ White
+
+* `~y~` Yellow
+
+* `~p~` Purple
+
+* `~l~` Black (lower case L)
+
+* `~h~` Turn text color lighter (used too much will make your text white, doesn't work on black)
+
+## `Special Text Letters`
+* `~u~` up arrow (gray)
+
+* `~d~` down arrow (gray)
+
+* `~<~` left arrow (gray)
+
+* `~>~` right arrow (gray)
+
+* `]` displays a * symbol (Only in text styles 3, 4 and 5)
+
+* `~k~` keyboard key mapping (e.g. ~k~~VEHICLE_TURRETLEFT~ and ~k~~PED_FIREWEAPON~). Look here for a list of keys
+
+:::caution
+Be careful, using too many text colors or special characters in one gametext may crash every player the gametext is shown to. Additionally, avoid using an uneven usage of the ~ character.
+Example: `~`~r~Hello, ~g~how are ~y~`~`you?`~`
+:::
+
+## `Text Styles`
+
+**Style 0**
+----------
+![![](/web/20190415201801im_/https://wiki.sa-mp.com/wroot/images2/thumb/0/00/Game-text-type0.jpg/120px-Game-text-type0.jpg)](https://web.archive.org/web/20190415201801im_/https://wiki.sa-mp.com/wroot/images2/thumb/0/00/Game-text-type0.jpg/120px-Game-text-type0.jpg)
```

Comment:
These are archive URLs, to add images drop them in the `static/images` directory and use `/images` as the base of the path.

Reply:
done